### ISSUE TITLE
Nomination title altered in admin view and in the sent email after aprouval

### ISSUE THREAD

**Issue by Sym-O** [opencredit/badgeos#453]
Hi ! 
Now our platform has been launched, I start fixing the encountered bugs. When someone has been nominated, The title of the nomination is badly display in 2 places : 
- when in admin / menu nomination : Title is correct, for example : "succes: I’m a writer !"
- when in admin / menu nomination / in the nomination itself : TItle has the problem, example : "succes: I&rsquo;m a writer !" . Interesting detail : If I correct myself the title and save the nomination, it is correctly save but the issue below remains
- when approving the nomination, an email is sent to the nominee : TItle is badly written again and even if we correct the title before approving nomination. 

We can see that this is not a European character problem here because our title are in English. No accent, but sometimes some apostrophe. 
Do you have any idea where I have to look to solve this issue ? I have a bunch of nominations to approved and want to solve this before. That would be a bad first contact if the first badge they received is badly written :)

Thanks !

**Comment by Sym-O** [comment]
Me again... I solved the issue and it seems it's related to wordpress and not badgeos. 
To keep trace of the solution, here it is.
Wordpress seems to have issues with the comma in the title. I found the solution here : http://emilegirard.com/dev/corriger-les-apostrophes-dans-wordpress.html
I adapted it to the specific code of my issue.
We just have to add the code section below  to the function.php.

----
function cleanApostrophes($content)
{
	$content = str_replace(array("'","`","’","&146;","&#2019;","&#8217;","&apos;","&amp;apos;","&#039;","&rsquo;"), "'", $content);
	return $content;
}
add_filter('the_title', 'cleanApostrophes');
add_filter('the_content', 'cleanApostrophes');
add_filter('the_excerpt', 'cleanApostrophes');
-----
